net-misc/curl/curl-9999.ebuild

```diff
@@ -305,30 +313,6 @@ multilib_src_configure() {
 		sed -i -e '/SUBDIRS/s:scripts::' Makefile || die
 	fi
 
-	# Fix up the pkg-config file to be more robust.
```

Comment:
What's this about? IIRC, it still needs to be there (@SoapGentoo and I discussed it recently, even).

Reply:
https://github.com/curl/curl/commit/f057de5a1a950a90d1920021db152a4b695f1a8a should eliminate our need for this based on what I've seen. I had to go looking for it as the live ebuild currently fails because the file doesn't exist.

Here's my current pkg-config:

```
Name: libcurl
URL: https://curl.se/
Description: Library to transfer files with ftp, http, etc.
Version: 8.9.0-DEV
Requires: 
Requires.private: zlib,openssl,libpsl,libnghttp2,nghttp3,libcares
Libs: -L${libdir} -lcurl 
Libs.private: -lcares -lnghttp3 -lnghttp2 -lpsl -lssl -lcrypto -lssl -lcrypto -lz
Cflags: -I${includedir} 
```